frontend/package.json

```diff
@@ -180,7 +180,7 @@
     "build:watch": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng build --watch --named-chunks",
     "tokens:generate": "theo src/app/spot/styles/tokens/tokens.yml --transform web --format sass,json --dest src/app/spot/styles/tokens/dist",
     "icon-font:generate": "node ./src/app/spot/icon-font/generate.js ./src/app/spot/icon-font",
-    "serve": "node --max_old_space_size=8096 ./node_modules/@angular/cli/bin/ng serve --host 0.0.0.0 --public-host http://localhost:4200",
```

Comment:
I added this piece of code to be able to set any exposed port to the frontend service. And of course have it configured only at only single place (`.env`). Do you see any problems with that approach? 

`.env.example` was amended of course

Reply:
I'd probably rather use `export` than `eval` but other than that, that's fine!